oak_hello_world_linux_init/src/init.rs

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2022 The Project Oak Authors
+// Copyright 2023 The Project Oak Authors
```

Comment:
When did that happen? :)

Reply:
I was still adding headers with last year's date last week :)